client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider-utils.tsx

```diff
@@ -0,0 +1,90 @@
+// TODO: Some stuff:
+// - Move this inline in "stats-purchase-tier-upgrade-slider.tsx" OR
+// - Add imports as needed
+// - Translate locally?
+
+type StatsPlanTier = {
+	id: string;
+	description: string;
+	price: string;
+	views: string;
+};
+
+// TODO: Remove hard-coded data once API is working.
+// We need to understand the shape of the data first.
+// Once we have that, replace this with a sample response object.
+const planTiers: StatsPlanTier[] = [
+	{
+		id: '10k',
+		price: '$9',
+		views: '10k',
+		description: '$9/month for 10k views',
+	},
+	{
+		id: '100k',
+		price: '$19',
+		views: '100k',
+		description: '$19/month for 100k views',
+	},
+	{
+		id: '250k',
+		price: '$29',
+		views: '250k',
+		description: '$29/month for 250k views',
+	},
+	{
+		id: '500k',
+		price: '$49',
+		views: '500k',
+		description: '$49/month for 500k views',
+	},
+	{
+		id: '1M',
+		price: '$69',
+		views: '1M',
+		description: '$69/month for 1M views',
+	},
+	{
+		id: '1M++',
+		price: '$69++',
+		views: '1M++',
+		description: '$25/month per million views if views exceed 1M',
+	},
+];
+
+function getPlanTiers(): StatsPlanTier[] {
+	// TODO: Work out data fetch mechanisim.
```

Comment:
I believe there is existing mechanism that pulls `/products` API which has the `price_tier_list`

Reply:
Added a GH Issue for API integration [here](https://github.com/orgs/Automattic/projects/484/views/18?groupedBy%5BcolumnId%5D=Status&sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=Status).